compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -280,6 +280,12 @@ object Parsers {
         syntaxError(msg, offset)
         skip(stopAtComma = true)
 
+    def syntaxErrorOrIncomplete(msg: Message, offset: Span): Unit =
```

Comment:
Not sure if this is best or not, but I overloaded the previous `syntaxErrorOrIncomplete` to also be able to take in a `Span` instead of just the offset.

Reply:
`Spic and Span`! but it would be more usual to call the param `span` as in `syntaxError`.

Reply:
ah whoops, good catch. Updated.